scripts/globusmonitor/globus_uploader_service.py

```diff
@@ -439,25 +460,37 @@ def notifyClowderOfCompletedTask(task):
                                        headers={'Content-Type':header},
                                        data=content)
 
-                        if fi.status_code in [500, 502, 504]:
+                        if fi.status_code in [104, 500, 502, 504]:
```

Comment:
I'm pretty sure the error here isn't an HTTP status, this is an error returned by the python sockets.

http://www.itmaybeahack.com/homepage/iblog/architecture/C551260341/E20081031204203/index.html